packages/@aws-cdk/aws-route53/README.md

```diff
@@ -88,6 +88,30 @@ Constructs are available for A, AAAA, CAA, CNAME, MX, NS, SRV and TXT records.
 Use the `CaaAmazonRecord` construct to easily restrict certificate authorities
 allowed to issue certificates for a domain to Amazon only.
 
+To add a NS record to a HostedZone in different account
+
+```ts
+import * as route53 from '@aws-cdk/aws-route53';
+
+// In the HostedZone containing a account
+const parentZone = new route53.PublicHostedZone(this, 'HostedZone', {
+  zoneName: 'someexample.com',
+  crossAccountZoneDelegationPrinciple: new iam.AccountPrincipal('12345678901')
+});
+
+// In this account
+const subZone = new route53.PublicHostedZone(this, 'SubZone', {
+  zoneName: 'sub.someexample.com'
+});
+
+new route53.CrossAccountZoneDelegationRecord(this, 'delegate', {
+  recordName: 'sub.someexample.com',
+  nameServers: subZone.hostedZoneNameServers!,
+  zoneName: 'someexample.com',
+  delegationRole: parentZone.crossAccountDelegationRole
+});
```

Comment:
Often, we opt for taking in a construct interface over the individual parts. Taking an `IHostedZone` here instead of the name servers and zone name would allow for some flexibility going forward (e.g., sending the zone ARN to the custom resource instead of the zone name).

The recordName could also be inferred by default from the hosted zone, which would lead to this nice usage:

```suggestion
new route53.CrossAccountZoneDelegationRecord(this, 'delegate', {
  delegatedZone: subZone,
  parentZone,
  delegationRole: parentZone.crossAccountDelegationRole
});
```

What do you think?

Reply:
This would make sense for the `delegatedZone` since you would be creating that wherever you would create the record. But for `parentZone` this would not be possible since it would not be created in this context and importing would not be an option.

Reply:
Added usage of `delegationZone` and `parentHostedZoneId`.